.github/workflows/main.yml

```diff
@@ -30,15 +30,15 @@ jobs:
       should-run-hindcast: ${{ steps.hindcast-logic.outputs.should-run-hindcast }}
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Check for changes in wave/io/hindcast
         id: changes
-        uses: dorny/paths-filter@v2
+        uses: dorny/paths-filter@v3
         with:
           filters: |
             wave_io_hindcast:
-              - 'mhkit/tests/wave/io/hindcast/**'
+              - 'mhkit/wave/io/hindcast/**'
```

Comment:
we should probably check for changes in both places: hindcast tests and the hindcast module itself

Reply:
Thanks I have been too lazy to investigate if I could just add the test line to the filters.  Looks like I can.

I'll push the commit up after the current actions pass or fail.